weaviate/connect/helpers.py

```diff
@@ -178,6 +184,12 @@ def connect_to_embedded(
             This includes many additional, rarely used config options. use wvc.init.AdditionalConfig() to configure.
         `version`
             Weaviate version to be used for the embedded instance.
+        `persistence_data_path`
+            Path to the directory where the embedded instance should store its data. If not provided, a temporary directory will be used.
```

Comment:
it will go to the default dir for application data, see here https://weaviate.io/developers/weaviate/installation/embedded#embedded-options